nix/marlowe-cardano/cardano-tools.nix

```diff
@@ -0,0 +1,15 @@
+{ repoRoot, inputs, pkgs, lib, system }:
+
+# TODO Remove this file once cardano-node can build aarch64-darwin.
+
+{
+  cardano-node =
+    if system == "aarch64-darwin"
```

Comment:
We can get native `aarch64-darwin` cardano-node and cardano-cli if you can upgrade from 8.1.2 to 8.2.1 (or greater)

Reply:
We cannot upgrade beyond 8.1.2 until IOG recommends that stake pool operators do that. The last time I looked, the SPOs were told to continue using 8.1.2.